### ISSUE TITLE
A11y_Azure Data Studio extension 'Azure SQL Migration'_Database backup_ScreenReader: Narrator is not announcing column header name while navigating using ctrl+alt+arrow keys in "target database" grid

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/azuredatastudio#19846]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### Github Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #Benchmark; #SH-AzureSQLMigration-Win32-Jun2022; #DesktopApp; #Win32; #SQL Migration Control Plane; #WCAG1.3.1; #ScreenReader; #Narrator;

### Environment and OS details:
Azure Data Studio Version 1.37.0(User Setup)
Windows Version 21H2 (OS Build 22000.675)

### Note:

- This issue is also observed for Databases grid in Databases for assessment step
- This issue is also observed for Authentication keys grid present in "Azure Database Migration service" step
- This issue is not observed with NVDA

### Repro Steps:

1. Turn on Narrator
2. Install Azure data studio app and install Azure SQL migration extension
3. Connect to SQL server, tab till respective server, right click and select manage
4. Tab till Azure SQL Migration in sitemap and press enter key
5. Tab till migrate to azure sql and press enter
6. Navigate to databases grid in "Databases for assessment" step
7. Select few databases and navigate to Database backup step by filling all mandatory fields in all steps
8. Navigate to target database grid and verify whether Narrator is not announcing column header name while navigating using ctrl+alt+arrow keys or not

### Actual Result:
Narrator is not announcing column header name while navigating using ctrl+alt+arrow keys in "target database" grid

### Expected Result:
Narrator should announce column header name while navigating using ctrl+alt+arrow keys in "target database" grid

### Attachment:
![Narrator_databasebackup_](https://user-images.githubusercontent.com/83570638/176171600-8d39455e-133a-4075-82fd-e64d87596662.gif)

**Comment by aasimkhan30** [comment]
This is a narrator issue and we do not optimize for specific screen-readers. Please close this bug

**Comment by aasimkhan30** [comment]
Closing this issue as it is a narrator specific bug.

**Comment by VijayaBharathyA** [comment]
@aasimkhan30 
Is it external to narrator could please provide a sample app we will be routing this bug to Narrator Team

**Comment by kupatkar99** [comment]
@aasimkhan30 As mentioned above bug is external to narrator Could you please provide us the stand alone sample app to route the bug to external team. thanks!

**Comment by VijayaBharathyA** [comment]
@aasimkhan30 
Is it external to narrator could please provide a sample app we will be routing this bug to Narrator Team?

**Comment by VijayaBharathyA** [comment]
#Closed;
Since issue not observed with NVDA we can close this bug since Azure Data studio is an electron app.
